### PR TITLE
Graph active percent of workflows workers

### PR DESCRIPTION
**Pull request by prime-time**
Jira:
https://clever.atlassian.net/browse/INFRA-2704

Overview:
This change adds new graphs that show average activity vs time. It is helpful to see graphs of both container count and worker activity in the same dashboard. `ActivityActivePercent` aka `active-percent` is a custom metric that is being used to autoscale workers.

Example of the new graphs:
screen shot 2017-12-15 at 10 39 43 am

### CODE REVIEW

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -729,7 +731,71 @@ var createWorkerRows = (function() {
...
+              "ActivityArn": "arn:aws:states:us-west-2:589690932525:activity:" + service.env_app
+            },
+            "function": "mean",
+            "metricName": "ActivityActivePercent",
+            "namespace": "StatesCustom",
+            "region": "us-west-1",
+            "statistics": [ "Average" ]
+          }
+        ],
+        "yaxes": [ { "min":0, "format":"short" }, {} ]
```
Comment (rgarcia) [inline dashboards/scripted_services.js]: is it possible to have a separate left and right axis here, one for active percent, one for container count? I fear with a single y-axis, the units won't line up nicely for low container counts

**Comment by prime-time** [comment]
How does this look?
screen shot 2017-12-15 at 1 10 07 pm

**Comment by rgarcia** [comment]
@prime-time looks good. Maybe adding both left and right axes (even if they're identical) on the top two charts would align things?

**Comment by prime-time** [comment]
It does look better when all the time axes are aligned.
screen shot 2017-12-15 at 3 44 48 pm

**Pull request merged**